### ISSUE TITLE
Does this work with CLIP guidance for stable diffusion?

### ISSUE THREAD

**Issue by eware-godaddy** [Jack000/glid-3-xl-stable#8]
Wondering if this code base means we can use CLIP guidance for generation instead of the classifier free guidance in the regular model?

**Comment by Jack000** [comment]
kind of, you can backprop the gradients through the vae but it uses a lot of vram and doesn't work that well in my experience. Ideally there should be a latent CLIP trained on the LDM embeddings instead of pixels.
